src/components/CommentItem.jsx

```diff
@@ -0,0 +1,12 @@
+const CommentItem = ({ comment }) => {
+
+  return (
+    <>
```

Comment:
It would be nice to contain each comment within a semantic tag, perhaps as a listitem?